docs/source/model_design/concept_model/ensemble_distributions/index.rst

```diff
@@ -204,20 +274,54 @@ specified.
 
   Make a table out of the above list, including the following possible columns:
 
-  * Name + Wikipedia link (or other source)
+  * Distribution name + Wikipedia link (or other source)
+  * Variable names for the distributions used in the R and Python repos
   * R function + documentation link
   * scipy.stats function + documentation link
   * Number of parameters for the distribution family (1, 2, or 3)
   * Formula for pdf or cdf?
-  * How to get distribution parameters from mean and variance?
+  * How to get distribution parameters from mean and variance (i.e. method of moments)?
+  * Tail behavior?
+  * Whether min and max are needed in addition to other parameters
+
+  It may be better to make a table with some small number of the above things,
+  then add a brief section on each distribution going into more details.
+
+  Also, here are some other things to add somewhere:
+
+    - Explain how the mean and variance will stay the same when we average the
+      distributions together because of the Laws of Total Expectation/Variance,
+      *unless* the weight for the Exponential distribution is nonzero, which
+      will throw the variance off.
+
+    - Add examples of GBD and Vivarium models that use ensemble distributions
```

Comment:
Yes to this one! This would be particularly nice, i think -- but maybe we can add this / link out as we document GBD models that use ensemble distributions? i think it's fine for now as a todo